<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tea_Check</class>
 <widget class="QWidget" name="Tea_Check">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:QLinearGradient(x1:0,y1:0,x2:0,y2:1,stop:0 #efefff,stop:1 #cfcfff);</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>661</width>
     <height>655</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>输入教授课程号点查询查看同学选课成绩</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>351</width>
      <height>411</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>班级</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>学生ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>学生名字</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>所属学院</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>成绩</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>课程号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sectionnumEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="dressButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>500</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>根据课程号筛选</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="sectiontableWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>281</width>
      <height>291</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>课程号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>课程名</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>点击查询课程看所教课程信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>480</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>查询课程</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>668</x>
     <y>11</y>
     <width>321</width>
     <height>655</height>
    </rect>
   </property>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>480</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>返回教师主界面</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;》》》》》》教师查看界面《《《《《《&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
